config/i18n.json

```diff
@@ -1125,10 +1125,10 @@
     "IndexedDBStorage": "Local Browser Storage",
     "LearnMore": "Learn more about DIM Sync",
     "MenuTitle": "Sync & Backups",
-    "ProfileErrorTitle": "DIM Sync Error",
-    "ProfileErrorBody": "We had a problem loading your data from DIM Sync. We'll keep trying to load it automatically.",
-    "UpdateErrorTitle": "DIM Sync Error",
-    "UpdateErrorBody": "We had a problem saving your data to DIM Sync. We'll automatically try again later.",
+    "ProfileErrorTitle": "DIM Sync Download Error",
+    "ProfileErrorBody": "We had a problem loading your latest settings, tags, loadouts, and searches from DIM Sync. Your data is still on our servers, and any updates you make locally will be saved when we can reconnect. We'll keep trying to load your data in the background as you use DIM.",
```

Comment:
this is presumably only from downtime right?
maximum reassurance might sound a little less like "there's a problem with your stuff" and more like "there's a problem communicating"

i would cover these points something like

- We had a problem communicating with DIM Sync.
- Your latest settings, tags, loadouts, and searches might not be shown.
- We'll keep trying to load your most recent data in the background.
- Changes you make now will be sent to DIM Sync when we can reconnect.